projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -758,16 +764,30 @@ public void cancelRunningCronJob(String teamName, String jobName, String executi
 
   public Set<String> listCronJobs() throws ApiException {
```

Comment:
So this function was never implemented properly? 
It always only supported v1Beta?

Reply:
Correct.